simpleflow/swf/executor.py

```diff
@@ -653,3 +652,24 @@ def fail(self, reason, details=None):
 
     def run(self, decision_response):
         return self.replay(decision_response)
+
+    def get_execution_context(self):
+        return self._execution_context
+
+    def build_execution_context(self, decision_response):
+        """
+        Extract data from the execution and history.
+        :param decision_response:
+        :type  decision_response: swf.responses.Response
+        """
+        execution = decision_response.execution
+        history = decision_response.history
+        workflow_started_event = history[0]
+        # The "if execution else None" are for tests that don't provide an execution object
```

Comment:
Why not
```python
if execution:
    self._execution_context = dict(...)
```
?

It's not exactly the same for tags, but well, if it's just tests and we miss all the other keys...